### PR TITLE
rotate_matrix.py: Add type hints for return values

### PR DESCRIPTION
**Pull request by cclauss**
@obelisk0114 Your review please?

### CODE REVIEW

File: matrix/rotate_matrix.py [hunk 1]
```diff
@@ -1,99 +1,76 @@
...
-    OR
-    
-    reverse_row(matrix)
-    transpose(matrix)
-    """
-    
-    return matrix
+
+    return reverse_column(transpose(matrix))
+    # OR.. transpose(reverse_column(matrix))
```
[the remark this answers is not in the document]
Comment (cclauss) [reply matrix/rotate_matrix.py]: My mistake...  Fix on the way.

**Comment by obelisk0114** [comment]
All codes are correct. 
(Option) You can add `transpose(reverse_column(matrix))` under `def rotate_90(matrix: [[]]) -> [[]]:`

**Comment by cclauss** [comment]
Added utility function __make_matrix()__.

**Comment by cclauss** [comment]
@brunohadlich Your review please?

**Comment by cclauss** [comment]
@obelisk0114 Can I please get you to re-review after the modifications.

**Pull request merged**